src/main/java/com/commercetools/sync/producttypes/helpers/ProductTypeSyncStatistics.java

```diff
@@ -0,0 +1,23 @@
+package com.commercetools.sync.producttypes.helpers;
+
+import com.commercetools.sync.commons.helpers.BaseSyncStatistics;
+
+import static java.lang.String.format;
+
+public class ProductTypeSyncStatistics extends BaseSyncStatistics {
```

Comment:
I created an integration test for that: https://github.com/commercetools/commercetools-sync-java/pull/296/files#diff-07965ad8d45b1a77e4bf0fc63ff109f8R393

Reply:
> We should have private constructor for this class, since we only expect it to be instantiated through the builder. Also we should make it final. I know this is not the case for the other *Statistics classes, but maybe we should change that.

Is there a builder for the *Statistics classes? or you mean that I have to develop one?

Reply:
@jortizsao 

> I created an integration test for that

I would say:
1. We should cover as much of the code (as possible) with unit tests.
2. a unit test is sufficient for the statistics since the integration tests have an overhead of the setting up/tearing down requests. So they are quite costly in the sense that they increase the time of the build significantly, etc..

see: https://martinfowler.com/bliki/TestPyramid.html

Reply:
> Is there a builder for the *Statistics classes? or you mean that I have to develop one?

Oh, my bad. I thought there is already one :( disregard the comment then. Just consider using `final`..

Reply:
> I would say:
> 
> We should cover as much of the code (as possible) with unit tests.
> a unit test is sufficient for the statistics since the integration tests have an overhead of the setting up/tearing down requests. So they are quite costly in the sense that they increase the time of the build significantly, etc..

Yes, I'm familiar with the test pyramid and I love unit tests (I usually develop using TDD when possible) and I prefer them over integration tests.

For the ProductTypeSync class and related classes and methods (basically the ones I've added in this PR and that don't create update actions), I found complex to add unit tests to these classes due to coupling/architecture, so I decided to follow the same existing approach (cover those test cases with integration tests ).

In this case is true that is possible to add a unit test for the `getReportMessage` method but in order to avoid a single test class for this unit test, I thought that the integration test was enough. 

Anyways, I add the unit test :)

Reply:
f2e2841c61d90f1f0cd0430767b6ee687b7a574d